packages/runtime/src/extensibility/facades/consumption/AttributesFacade.ts

```diff
@@ -177,10 +177,19 @@ export class AttributesFacade {
         return await this.deletePeerSharedAttributeAndNotifyOwnerUseCase.execute(request);
     }
 
+    public async deleteThirdPartyRelationshipAttributeAndNotifyPeer(
+        request: DeleteThirdPartyRelationshipAttributeAndNotifyPeerRequest
+    ): Promise<Result<DeleteThirdPartyRelationshipAttributeAndNotifyPeerResponse>> {
+        return await this.deleteThirdPartyRelationshipAttributeAndNotifyPeerUseCase.execute(request);
+    }
+
+    /**
+     * @deprecated deleteThirdPartyOwnedRelationshipAttributeAndNotifyPeer will be removed in the next major version. Use deleteThirdPartyRelationshipAttributeAndNotifyPeer instead.
```

Comment:
```suggestion
     * @deprecated use deleteThirdPartyRelationshipAttributeAndNotifyPeer instead.
```

we don't have to explain deprecations